data/relex-semantic.algs

```diff
@@ -534,7 +570,7 @@ PREDADJ_PREDET
 <F_L predet> != %
 <F_R subj links _predadj> != %
 =
-<F_R subj links _predadj> = <F_L predet>

```

Comment:
I cannot telll from here if you removed all uses of predet or not .. in any case, please make sure the wiki page that lists all of these relations is accurate and up-to-date.  (In particular, I think the back-n-forth confusion about quantity messed up the wiki page)